includes/dart-integrations/firebase-remote-config.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Firebase Remote Config
+description: "Learn more about the Sentry Firebase Remote Config integration for the Dart SDK."
+sidebar_order: 6
+platforms:
+  - flutter
+---
+
+The `sentry_firebase_remote_config` integration provides [Firebase Remote Config](https://firebase.google.com/docs/remote-config/get-started?platform=flutter) support for Sentry thus providing insight into feature flag evaluations.
+
+## Behavior
+
+- Adding the `SentryFirebaseRemoteConfig` integration will automatically track feature flag evaluations for boolean Firebase Remote Config values.
+- The integration is available in <PlatformLink to="/">Sentry SDK</PlatformLink> version 9.0.0 or higher.
+
+## Prerequisites
+
+1. The Sentry [Dart](/platforms/dart/#configure) SDK is initialized.
```

Comment:
```suggestion
1. SDK version `9.0.0` or higher is installed.

```

Reply:
I know we mention the initialized a couple times in other integrations. I'm actually thinking of removing them as imo they dont bring a lot of value